internals/secrethub/env_test.go

```diff
@@ -44,9 +41,7 @@ func TestParseEnvFile(t *testing.T) {
 			expected: nil,
 			errcheck: func(t *testing.T, err error) {
 				t.Helper()
-				if !strings.Contains(err.Error(), "yaml: unmarshal errors:") {
-					t.Errorf("unexpected error: %v (actual) != yaml: unmarshal errors (expected)", err)
-				}
+				assert.Equal(t, err, ErrTemplate(1, "template is not formatted as key=value pairs"))
```

Comment:
We're mixing some tests here. What about splitting it into 3 separate tests:
`TestParseYML`: to test the `.yml` parsing.
`TestParseEnv`: to test the `.env` testing.
`TestNewEnv`: to test whether the correct choice between `.yml` and `.env` is made.

Reply:
This is not yet done, right?

Reply:
Yes, you're right. The tests in `run_test.go` should still be split up.